src/effects/deeplearning/DeepModelManager.h

```diff
@@ -0,0 +1,109 @@
+/**********************************************************************
+
+   Audacity: A Digital Audio Editor
+   Audacity(R) is copyright (c) 1999-2021 Audacity Team.
+   License: GPL v2.  See License.txt.
+
+   DeepModelManager.h
+   Hugo Flores Garcia
+
+******************************************************************/
+/**
+
+\class DeepModelManager
+\brief tools for downloading and managing Audacity models hosted in HuggingFace
+
+TODO: add a more thorough description
+
+*/
+/*******************************************************************/
+
+#pragma once
+
+#include "DeepModel.h"
+#include "ModelCard.h"
+
+#include "Request.h"
+#include "NetworkManager.h"
+#include "widgets/ProgressDialog.h"
+
+class ModelManagerException : public std::exception
```

Comment:
The same about the base

Reply:
fixed [here](https://github.com/audacity/audacity/pull/1384/commits/617ff8699525ce6f628803d10d2fe0459d3c8fdc)